daml-lf/validation/src/main/scala/com/digitalasset/daml/lf/validation/Typing.scala

```diff
@@ -232,6 +232,15 @@ private[validation] object Typing {
         TForall(
           alpha.name -> KStar,
           TForall(beta.name -> KStar, TContractId(alpha) ->: TContractId(beta))),
+      // Unstable text functions
+      BTextAsciiToUpper -> (TText ->: TText),
+      BTextAsciiToLower -> (TText ->: TText),
+      BTextSliceIndex -> (TInt64 ->: TInt64 ->: TText ->: TText),
```

Comment:
Why do we have `BTextSliceIndex` twice with different types? I guess the first one should be `BTextSlice`?

Reply:
Indeed! Thanks!